modules/ROOT/pages/6.8-release-notes.adoc

```diff
@@ -337,6 +337,11 @@ As a consequence, the plugin failed to detect if the selection was at the beginn
 
 === A closing notification brings focus to the editor even when focus is outside of the editor
 // #TINY-10282
+Previously {productname}'s notification system would return focus back to the editor UI even in situations when the user "clicked" on the close button, and in situations when the `focus` was external to the editor UI.
```

Comment:
A small correction here. It wouldn't bring the focus to the UI but to the content ( the idea being that the user was typing, saw a notification, closed it, and could them immediately resume typing without having to click again. As far as I have understood ).
I'm also not sure what "would return focus back to the editor UI even in situations when the user "clicked" on the close button" is quite supposed to mean. The issue was a notification appearing and then timing out, which would snap focus from anywhere to the content when the notification closed.
The rest seems right.

Reply:
```suggestion
Previously {productname}'s notification system would bring the focus to the editors content after the user;

* inserted new content by typing;
* was then displayed a notification;
* closed it; then
* immediately resumed typing without having to click again

As a consequence, the notification would appear and then time out, which shifted the focus from any location (including when the user was focused external to the editor UI) back to the content when the notification closed.
```